doc/lua_api.txt

```diff
@@ -6221,8 +6221,8 @@ object you are working with still exists.
     * `time_from_last_punch` = time since last punch action of the puncher
     * `direction`: can be `nil`
 * `right_click(clicker)`; `clicker` is another `ObjectRef`
-* `get_hp()`: returns number of hitpoints (2 * number of hearts)
-* `set_hp(hp, reason)`: set number of hitpoints (2 * number of hearts).
+* `get_hp()`: returns number of health points (2 * number of hearts)
+* `set_hp(hp, reason)`: set number of health points (2 * number of hearts).
```

Comment:
`(2 * number of hearts)` is no longer correct due to customizable maximal HP values.